### PR TITLE
DD17 - Parameter Changes Monitor

### PR DESCRIPTION
**Pull request by RCantu92**
# Parameter Changes Monitor Bot

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New Agent
- [ ] Fix an agent error
- [ ] Update an agent logic/test/documentation
- [ ] Code style update (formatting, renaming)
- [ ] Other (please describe):

## New Agent PR readiness

- [x] `npm install` command works (i.e. package-lock is updated properly)
- [x] The `name` and `description` fields of `package.json` describe the agent properly
- [x] All the libraries listed in `package.json` are being used
- [x] All the development libraries are in `devDependencies` section of `package.json`
- [x] The agent README is updated
- [x] The agent README sufficiently describes what the agent does
- [x] All the alerts that the agent emits are listed in the README
- [x] The txn/block examples listed in the README generate the expected findings
- [x] Code is easy to follow and/or properly commented
- [x] Agent contains tests
- [x] All tests pass when running `npm test`
- [x] Tests are not making real network calls (i.e. network interaction is properly mocked)
- [x] Tests cover all the paths (i.e. negative & positive test cases)
- [x] When the tests complete, no warning/errors are shown
- [x] `npm start` works without errors when executed using a jsonRpcUrl for each of the networks specified in the README
- [x] The networks specified in the README and the networks are listed in the `package.json` are the same
- [x] The code does what the README describes
- [ ] Network calls are properly being cached (if duplicated calls can occur)
- [ ] A performance review has been done (e.g. `Promise.all` is used whenever possible)
- [x] The findings fields are filled out correctly
- [x] Findings `protocol` is specified (if the agent is for a specific one)
- [x] `metadata` of the findings contains all the information needed.
- [x] The SDK methods are being used appropriately (e.g. `filterLog`, `filterFunction`, `getEthersProvider`)
- [x] `ABI`s match the contracts used / `args` used from `LogDescriptions` exists in the `ABI`s
- [x] The agent follows Forta recommended [best practices](https://docs.forta.network/en/latest/best-practices/)
- [ ] Agent can recover from failed error calls when they can fail (e.g. errors are being caught on calls that can fail)
- [x] `repository` is added in the `package.json`

## Comments (optional)

If there is some useful information or important note for reviewing/understanding this agent please describe it here.

### CODE REVIEW

**Review by e1Ru1o — APPROVED** [review]
Please add automatic network data management using initialize function

File: DYDX-Bots/Parameter_Changes_Monitor/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "dydx-17",
+  "version": "0.0.1",
+  "description": "Monitor parameter changes",
```
Comment (yasmine9) [inline DYDX-Bots/Parameter_Changes_Monitor/package.json:4]:
```suggestion
  "description": "Monitor parameter changes in dYdX Safety and Liquidity Modules",
```
a suggestion to have a more detailed description.

**Review by yasmine9 — APPROVED** [review]
Great Job! nothing to mention other than the minor comments left. Same question from DD16 review about the networkManager type applies here.

**Review by yasmine9 — COMMENTED** [review]
PR can be merged after adding the error handling in network manager.

**Pull request merged**